ledger/complete/wal/checkpointer.go

```diff
@@ -776,6 +782,48 @@ func readCheckpointV5(f *os.File) ([]*trie.MTrie, error) {
 	return tries, nil
 }
 
+// ReadLastTrieRootHashFromCheckpoint returns last trie's root hash from checkpoint file f.
+func ReadLastTrieRootHashFromCheckpoint(f *os.File) (hash.Hash, error) {
```

Comment:
could we please include a short documentation of the returned error. Something like the following would already be sufficient:

```suggestion

// All returned errors indicate that the given checkpoint file is either corrupted or incompatible. As the function is side-effect free, all failures are simply a no-op.
func ReadLastTrieRootHashFromCheckpoint(f *os.File) (hash.Hash, error) {
```